ConwaysGameOfLife.Tests/UniverseHelper.cs

```diff
@@ -61,11 +61,11 @@ public static bool ListsOfCellsAreEqual(List<Cell> obj1, List<Cell> obj2)
 
         public static bool LocationsAreEqual(Location obj1, Location obj2)
         {
-            if (obj1.X != obj2.X && obj1.Y != obj2.Y)
+            if (obj1.X == obj2.X && obj1.Y == obj2.Y)

```

Comment:
good job removing the double negative.

could replace obj1 and obj2 with better names now that we're using custom helpers instead of overrides (can no longer just pass in any old object). this goes for all of the equality helpers

```suggestion
            if (location1.X == location2.X && location1.Y == location2.Y)

```